From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jboeshart__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ccompy committed:
Merge pull request #10 from MicrosoftDocs/master

update ase docs

Commit:
Deleting file as it is no longer needed.

david-stanford committed:
[ACI] Set environment variables

ccompy committed:
Merge pull request #11 from MicrosoftDocs/master

sync to master

david-stanford committed:
fix format of tabbing

david-stanford committed:
remove tabs, add correct powershell commands

MarkMcGeeAtAquent committed:
Merge pull request #14 from MicrosoftDocs/master

 sync master

MarkMcGeeAtAquent committed:
Removed 'en-us'

MarkMcGeeAtAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

MarkMcGeeAtAquent committed:
commit

ccompy committed:
update forced tunnel doc

MarkMcGeeAtAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

ccompy committed:
update forced tunnel doc

barbkess committed:
wwi

barbkess committed:
load wwi

barbkess committed:
updates

barbkess committed:
updated pngs

barbkess committed:
firewall pngs

barbkess committed:
latest

barbkess committed:
updates

ccompy committed:
update

ccompy committed:
doc update

ccompy committed:
images

ccompy committed:
another update

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into wwi

MarkMcGeeAtAquent committed:
Merge branch 'master' of https://github.com/MarkMcGeeAtAquent/azure-docs-pr

MarkMcGeeAtAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

barbkess committed:
fixes

ccompy committed:
update forced tunnel

ccompy committed:
text updates

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into wwi

barbkess committed:
create statistics

barbkess committed:
statistics

mimig1 committed:
Cosmos DB: Updated title, H1, and description of SQL API QS

MarkMcGeeAtAquent committed:
updates

ccompy committed:
another update

dlepow committed:
N-series: LIS update

ccompy committed:
changed force to forced

swkrish committed:
Create active-directory-aadconnect-pass-through-authentication-gdpr

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-gdpr

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-gdpr

swkrish committed:
Rename active-directory-aadconnect-pass-through-authentication-gdpr to active-directory-aadconnect-pass-through-authentication-gdpr.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-gdpr.md

swkrish committed:
Create active-directory-sso-gdpr.md

swkrish committed:
Rename active-directory-sso-gdpr.md to active-directory-aadconnect-sso-gdpr.md

swkrish committed:
Update active-directory-aadconnect-sso-quick-start.md

paulathurman committed:
edit pass: site-recovery-batch-2-of-2-new

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into wwi

barbkess committed:
add TOC change

barbkess committed:
fix script

j-martens committed:
Update .openpublishing.redirection.json

Adding redirection from articles/machine-learning/studio/datamarket-deprecation.md    --- to: https://microsoft.com/cognitive

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

barbkess committed:
fix drop statements

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

swkrish committed:
Merge branch 'master' into master

v-viinde committed:
USER STORY 1206266 SaaS App Tutorial: LoginRadius Tutorial Update

Commit:
quickfix12

MikeDodaro committed:
missing metadata corrupting build

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-gdpr.md

swkrish committed:
Update TOC.md

swkrish committed:
Update active-directory-aadconnect-sso-gdpr.md

ccompy committed:
tweaks

PRMerger13 committed:
Merge pull request #35191 from MarkusVi/quickfix12

quickfix12

dlepow committed:
Updated LIS download link

christiankuhtz committed:
tweak

PRMerger16 committed:
Merge pull request #35120 from MarkMcGeeAtAquent/table-c++

Cosmos DB: updates for C++/Table API

MarkMcGeeAtAquent committed:
incorp feedback

rloutlaw committed:
small changes from Java CDA review

PRMerger18 committed:
Merge pull request #35198 from christiankuhtz/patch-70

tweak

PRMerger18 committed:
Merge pull request #35127 from dlepow/fix2

Linux N-series VMs: LIS update

barbkess committed:
fix reviewers remarks

ShawnJackson committed:
Update vmware-azure-failback.md

Lisaco88 committed:
Fixed style issues

ccompy committed:
tweak

j-martens committed:
Merge branch 'patch-13' of https://github.com/LuisCabrer/azure-docs-pr into jmartens-dev

# Conflicts:
#	articles/machine-learning/studio/datamarket-deprecation.md

ShawnJackson committed:
Update vmware-azure-troubleshoot-failback-reprotect.md

Lisaco88 committed:
Fixed 2 typos

msmbaldwin committed:
Updated author

david-stanford committed:
Update text, add change to the TOC

roygara committed:
Updating link name.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into phpqs

j-martens committed:
Merge branch 'master' of https://github.com/j-martens/azure-docs-pr
attempting to fix merge conflicts

ShawnJackson committed:
Update vmware-physical-azure-support-matrix.md

Commit:
Merge pull request #35202 from MarkMcGeeAtAquent/table-c++

Cosmos DB: incorp feedback

MikeDodaro committed:
Fix code tics

Commit:
namedllocations02

david-stanford committed:
updated grammar/style

mattbriggs committed:
cmdlet deprecated

PRMerger12 committed:
Merge pull request #35200 from rloutlaw/rloutlaw/cosmos-db-cda-review

small changes from Java CDA review

PRMerger13 committed:
Merge pull request #35204 from roygara/phpqs

Changing the wording on a link.

bandersmsft committed:
policy - cli quickstart update

v-geberr committed:
fixing TOC for SDKs and APIs

Lisaco88 committed:
Merge pull request #34711 from barbkess/wwi

Wwi

ShawnJackson committed:
Merge pull request #35157 from paulathurman/site-recovery-batch-2-of-2-new

edit pass: site-recovery-batch-2-of-2-new

PRMerger17 committed:
Merge pull request #35205 from bandersmsft/master

policy - cli quickstart update

Lisaco88 committed:
Merge pull request #35203 from msmbaldwin/marketowner

Updated author

barclayn committed:
added en-us to the portal.msrc.microsoft.com because it is not localized and it will not go to the form otherwise

PRMerger19 committed:
Merge pull request #35208 from MarkusVi/namedllocations02

namedllocations02

Commit:
Merge pull request #35210 from barclayn/pen-test

added en-us to the portal.msrc.microsoft.com because it is not locali…

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-gdpr.md

MikeDodaro committed:
Fix code end tics

Commit:
Merge pull request #35092 from mimig1/patch-221

Cosmos DB: Updated title, H1, and description of SQL API QS

tfitzmac committed:
updated tag supported characters

PRMerger14 committed:
Merge pull request #35196 from MikeDodaro/fixMissingMetaDataNews

missing metadata corrupting build

mrbullwinkle committed:
ISO8601 issue

karthikananth committed:
Update expressroute-locations.md

karthikananth committed:
Update expressroute-locations.md

v-shils committed:
Merge pull request #35207 from mattbriggs/mdb_2158606

cmdlet deprecated

PRMerger18 committed:
Merge pull request #35216 from mrbullwinkle/mrb_13_2018_snapshot_take2

ISO8601 issue

Lisaco88 committed:
Merge pull request #35168 from swkrish/master

GDRP docs + fixes based on customer feedback

v-shils committed:
Merge pull request #35215 from tfitzmac/0312tags

updated tag supported characters

mattbriggs committed:
updating sentence and date to this year

Commit:
Merge pull request #35217 from karthikananth/patch-27

Update expressroute-locations.md

Lisaco88 committed:
Merge pull request #35206 from v-geberr/0313-fix-sdk-link

fixing TOC for SDKs and APIs

Commit:
minor fix to link to service fabric docs

PRMerger16 committed:
Merge pull request #35226 from mattbriggs/mdb_manage_storage

updating sentence and date to this year

ktoliver committed:
Merge pull request #35177 from v-viinde/loginradiusupdate

USER STORY 1206266 SaaS App Tutorial: LoginRadius Tutorial Update

PRMerger18 committed:
Merge pull request #35225 from dkkapur/master

minor fix to link to service fabric docs

christiankuhtz committed:
tweak

j-martens committed:
redirecting /studio/datamarket-deprecation.md

Commit:
Merge pull request #35230 from christiankuhtz/patch-73

tweak

Commit:
adding deployment info about containers in a containerGroup and fixing region limit

Ja-Dunn committed:
Merge pull request #35235 from j-martens/jmartens-dev

AML: Deleting and redirecting articles/machine-learning/studio/datamarket-deprecation.md

j-martens committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#5681

PRMerger12 committed:
Merge pull request #35237 from jluk/master

adding deployment info about containers in a containerGroup and fixin…

ccompy committed:
doc update

removed ARM and say resource manager

j-martens committed:
AML: deleting duplicate entry

christiankuhtz committed:
tweak

PRMerger16 committed:
Merge pull request #35239 from tfitzmac/0313crossrg

updated note on resource group limitations

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Ja-Dunn committed:
Merge pull request #34298 from david-stanford/aci-env-vars

[ACI] Set environment variables

Ja-Dunn committed:
Merge pull request #34901 from ccompy/mar1update

updates to the forced tunnel doc

PRMerger19 committed:
Merge pull request #35241 from christiankuhtz/patch-74

tweak

v-alje committed:
Merge pull request #35242 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #35238 from j-martens/master

AML: Cleaning up branch
